src/main/java/frc/robot/constants/swerve/DriveConstants.java

```diff
@@ -115,16 +115,23 @@ public class DriveConstants {
   public static double kHeadingP= 1;//4
   public static double kHeadingD= 0;
 
-   //balance PID
-   public static double kBalanceP= 0.01;
-   public static double kBalanceI= 0;
-   public static double kBalanceD= 0;
-   public static final double kBalanceMaxOutput = 0.5;
-   public static final double kBalanceTolerance = 0.1;
+  //balance PID
+  public static double kBalanceP= 0.05;
+  public static double kBalanceI= 0;
+  public static double kBalanceD= 0;
+  public static final double kBalanceMaxOutput = 0.5;
+  public static final double kBalanceTolerance = 2;
 
-   //balance timer
-   public static double kBalanceEndTime = 0.8;
-   public static double kBalanceStartTime = 0.5;
+  //balance timer
```

Comment:
Nice providing comments for this not obvious logic! Does it makes sense to move these comments inside the BalanceCommand.java to make them  easier to find?